File: ScalaExamples/src/Arcade/Intro/TheJourneyBegins.scala
package Arcade.Intro

class TheJourneyBegins {

	def add(param1: Int, param2: Int): Int = {
		param1 + param2;
	}

	def centuryFromYear(year: Int): Int = {
		(year - 1) / 100 + 1
	}

	def checkPalindrome(inputString: String): Boolean = {
		inputString.reverse equals inputString
	}

}

File: CodeProjects/Arcade/TheJourneyBegins.cs
using System;

namespace CodeProjects
{
	class TheJourneyBegins
	{
		int centuryFromYear(int year)
		{
			int result = year / 100;

			return year % 100 == 0 ? result : result + 1;
		}

		bool checkPalindrome(string inputString)
		{
			int j = inputString.Length;

			for (int i = 0; i < inputString.Length; i++)
			{
				j--;
				if (inputString[i] != inputString[j])
				{
					return false;
				}
			}

			return true;
		}
	}
}
